CHANGELOG.md

```diff
@@ -98,6 +98,7 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 - Fixed [#535](https://github.com/JabRef/jabref/issues/535): Add merge action to right click menu
 - Fixed [#1115](https://github.com/JabRef/jabref/issues/1115): Wrong warning message when importing duplicate entries
 - Fixed [#935](https://github.com/JabRef/jabref/issues/935): PDFs, which are readable, but carry a protection for editing, are treated by the XMP parser and the importer generating a BibTeX entry based on the content.
+- now the Preview Panel will be shown with a single click directly after startup
```

Comment:
done, and I would definitely say that this is a bug (look at the steps to reproduce in the first post).